src/core/connectors.ts

```diff
@@ -1719,10 +1719,10 @@ export default <ConnectorMeta[]>[
 		id: 'coderadio',
 	},
 	{
-		label: 'Dash Radio',
-		matches: ['*://dashradio.com/*'],
-		js: 'dashradio.js',
-		id: 'dashradio',
+		label: 'LITT Live',
+		matches: ['*://littlive.com/*'],
+		js: 'littlive.js',
+		id: 'littlive',
```

Comment:
With this ID change we would lose configuration changes with the old one. Don't think it'll be much on an issue though